src/Layer/OrientedImageLayer.js

```diff
@@ -147,14 +149,15 @@ class OrientedImageLayer extends GeometryLayer {
                 pano.id = pano.geometry[0].properties.id;
                 pano.index = i++;
             }
-        }).then(() => {
+        }, () => reject(new Error('oriented image layer: failed parse orientation'))).then(() => {
```

Comment:
The rejection doesn't occur on the failed parsing of the orientation configuration, but rather on the `source.whenReady` giving a rejection. This rejection should be the next one, rather than a "failed parse calibration", which should be removed as well.

Reply:
You are right. We'll have to do a PR to handle the errors properly.